epinar/assignment3/word_search.py

```diff
@@ -0,0 +1,128 @@
+import string
+import unittest
+
+
+class Dictionary:
+
+	def __init__(self, words, prefixes):
+		"""
+		The dictionary class to keep the values.
+		:param words: Valid entries in a dictionary.
```

Comment:
can you use the Python type annotations instead of these syntax for the function signature?

https://docs.python.org/3/library/typing.html